src/app/components/navigation/main-navigation.component.ts

```diff
@@ -34,32 +34,29 @@ export class MainNavigationComponent implements OnInit, OnDestroy {
 
   isHandset$: Observable<boolean> = this.breakpointObserver
     .observe([Breakpoints.Handset, Breakpoints.Tablet])
-    .pipe(map(result => result.matches));
-
-  constructor(private breakpointObserver: BreakpointObserver, private router: Router,
-    public dialog: MatDialog) {
-    this.routerSubscription = this.router.events
-      .pipe(filter(event => event instanceof NavigationStart))
-      .subscribe(() => {
-        if (this.drawer.mode !== 'side' && this.drawer.opened) {
-          this.drawer.close();
-        }
-      });
+    .pipe(map((result) => result.matches));
+
+  constructor(
+    private breakpointObserver: BreakpointObserver,
+    private router: Router,
+    public dialog: MatDialog,
+  ) {
+    this.routerSubscription = this.router.events.pipe(filter((event) => event instanceof NavigationStart)).subscribe(() => {
```

Comment:
it's more readable to have line breaks after pipe

Reply:
Comes from prettier